web/src/services/Analytics/Analytics.service.ts

```diff
@@ -11,11 +11,9 @@ type TAnalyticsService = {
   event<A>(category: Categories, action: A, label: string): void;
 };
 
-const initializePromise = instance.initialize();
```

Comment:
We were initializing the analytics instance always. We already have the logic to initialize it (depending on the `analyticsEnabled` flag) in the `AnalyticsProvider`.

Reply:
I think I tried doing this to debug a problem with our integration tests. Great catch @jorgeepc